src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script
+      type="text/javascript"
+      src="./main.js"
+    >
+    </script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="userSurname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+          <input
+            type="text"
+            name="userName"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          How old are You?
+          <input
+            type="number"
+            name="userAge"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form__field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="acceptAgreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="form__fieldShort">
+          E-mail:
+          <input
+            type="email"
+            name="userEmail"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="userPassword"
+            minlength="8"
+            maxlength="24"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label>
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCat"
+              value="yes"
+            > Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCat"
+              value="no"
+            > No
+          </label>
+        </label>
```

Comment:
1) no needs to wrap Do you love cats? into a label
2) text and input should be on the same level
```suggestion
        <div>
          Do you love cats?
          <label>
            <input
              type="radio"
              name="loveCat"
              value="yes"
            > 
            Yes
          </label>

          <label>
            <input
              type="radio"
              name="loveCat"
              value="no"
            > 
            No
          </label>
        </div>
```